src/main/java/org/assertj/jodatime/api/DateTimeAssert.java

```diff
@@ -51,6 +68,139 @@ protected DateTime getActual() {
     return actual;
   }
 
+  /**
+   * Verifies that the year of the actual {@code DateTime} is equal to the given year
+   * <p>
+   * Example :
+   * <pre><code class='java'> assertThat(new DateTime(&quot;2000-01-01&quot;)).hasYear(1);</code></pre>
+   *
+   * @param other the given year.
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code DateTime} is {@code null}.
+   * @throws IllegalArgumentException if other {@code DateTime} is {@code null}.
+   * @throws AssertionError if the year of the actual {@code DateTime} is not equal to the given year.
+   */
+  public DateTimeAssert hasYear(Integer other){
```

Comment:
Any reason for using an `Integer` in place of an `int` ?  that would spare us to check for a null parameter.

Reply:
No reason, just didn't think about that.  I'll change it to an int. I like that suggestion.